src/mappers/provinces/province_mapper_importer.h

```diff
@@ -22,7 +23,8 @@ namespace mappers
 class ProvinceMapperImporter
 {
   public:
-   ProvinceMapperImporter(const commonItems::ModFilesystem& filesystem);
+   ProvinceMapperImporter(const commonItems::ModFilesystem& filesystem,
+       const std::optional<std::map<std::string, vic3::StateRegion>>& vic3_state_regions);
```

Comment:
In regards to the optional, containers naturally have the 'empty' state, which you should be able to use in the tests with either `{}` or `std::map<std::string, vic3::StateRegion{}>` if it can't deduce the type. Optional is extraneous with a map.

Reply:
fix with [remove std:optional](https://github.com/ParadoxGameConverters/Vic3ToHoI4/pull/632/commits/e9f49f0878acba836e5182e705d7dbea9e00a2bd)